chains/evm/voter/voter.go

```diff
@@ -61,57 +46,22 @@ func (w *EVMVoter) VoteProposal(m *relayer.Message) error {
 	if err != nil {
 		return err
 	}
-	ps, err := prop.Status(w.client)
-	if err != nil {
-		log.Error().Err(err).Msgf("error getting proposal status %+v", prop)
-	}
-
-	votedByCurrentExecutor, err := prop.VotedBy(w.client, w.client.RelayerAddress())
+	ps, err := calls.ProposalStatus(w.client, prop)
```

Comment:
This is related to L44, method name `VoteProposal` could be renamed so it is clearer (ie, `VoteOnProposal`) and that it demonstrates a state change. 

Returning just an error is a bit confusing as to what this method does, and just by looking at the function signature along with the name it isn't clear whether the method just reads data or if it actually changes data.

Only after reading the function body do you realize it actually makes a contract call